analyses.qmd

```diff
@@ -925,10 +936,19 @@ There are zeros in the data. Is that a mistake?
 We could try and aggregate the data first, before modelling.
 
 ### Data aggregation
-
+We can try to sum the number of stems per LocalityCode.
+This variation was explained in the random structure in the models above.
+If we aggregate this way, the data might fit easier.
 ```{r}
 modelData2 <- modelData |>
-  group_by(Taxa, Region, LocalityName, LocalityCode, Treatment, productivityIndex) |>
+  group_by(Taxa, 
+           Region, 
+           LocalityName, 
+           LocalityCode, 
+           Treatment, 
+           productivityIndex,
+           HeightClass,
```

Comment:
I added HeightClass and height_cm here so that these variable become part of modelData2. Note that I added rows with Quantity = 0 for all 'missing cases', i.e. I inferred absence data. These zeros have HieghtClass = NA, but in reality the zero apåplies to all height classes (the species does not exist in that plot)

Reply:
@gunnaraustrheim